Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -194,11 +194,20 @@ void CodeViewWidget::FontBasedSizing()
   constexpr int extra_text_width = 8;
 
   const QFontMetrics fm(Settings::Instance().GetDebugFont());
+
+  const auto width = [&fm](QString text) {
```

Comment:
Should this go into QtUtils? It looks like this function could be useful in other cases. (Also, any reason for passing text by value?)

Reply:
> Should this go into QtUtils? It looks like this function could be useful in other cases.

It doesn't seem like we are using this anywhere else. Maybe we should move it if we decide to use this in more places.

> (Also, any reason for passing text by value?)

No particular reason. I suppose it's neither less efficient nor more efficient (at least by any measurable amount).